backend/docker-compose_init.bat

```diff
@@ -0,0 +1,3 @@
+@echo off

```

Comment:
I don't see a command to build the project with official maven docker image before running compose